stdlib/2/os/__init__.pyi

```diff
@@ -10,7 +10,7 @@ from mypy_extensions import NoReturn
 error = OSError
 name = ...  # type: str
 
-class _Environ(MutableMapping[str, str]):
+class _Environ(MutableMapping[Union[str, unicode], Union[str, unicode]]):
```

Comment:
Ah OK, I see what you mean. I thought `os.environ` was more akin to an ordinary dict, but it does implicit encoding even in python 2:
```python
>>> import os
>>> os.environ['💣'] = '🔥'
>>> type(os.environ['💣'])
<type 'str'>
>>> os.environ['💣']
'\xf0\x9f\x94\xa5'
```